src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -593,5 +608,4 @@ public static boolean isCodePointWhitespace(int[] codePoints, int index) {
         final char character = Character.toChars(codePoints[index])[0];
         return Character.isWhitespace(character);
     }
-
```

Comment:
please restore it, we do not have Check for this, but all other classes have it, I do not remember why, but I like this extra space.

Reply:
done